state-chain/runtime/src/chainflip/bitcoin_elections.rs

```diff
@@ -357,26 +456,32 @@ impl
 		BitcoinBlockHeightTrackingES,
 		BitcoinDepositChannelWitnessingES,
 		BitcoinVaultDepositWitnessingES,
+		BitcoinEgressWitnessingES,
 		BitcoinLiveness,
 	> for BitcoinElectionHooks
 {
 	fn on_finalize(
-		(block_height_tracking_identifiers, deposit_channel_witnessing_identifiers, vault_deposits_identifiers, liveness_identifiers): (
+		(block_height_tracking_identifiers, deposit_channel_witnessing_identifiers, vault_deposits_identifiers, egress_identifiers,  liveness_identifiers): (
 			Vec<
 				ElectionIdentifier<
-					<BitcoinBlockHeightTrackingES as ElectoralSystemTypes>::ElectionIdentifierExtra,
+					<BitcoinDepositChannelWitnessingES as ElectoralSystemTypes>::ElectionIdentifierExtra,
 				>,
 			>,
 			Vec<
 				ElectionIdentifier<
-					<BitcoinDepositChannelWitnessingES as ElectoralSystemTypes>::ElectionIdentifierExtra,
+					<BitcoinVaultDepositWitnessingES as ElectoralSystemTypes>::ElectionIdentifierExtra,
```

Comment:
It's interesting that this works apparently, but it seems like we now have two `BitcoinVaultDepositWitnessingES` and no `DepositChannelWitnessingES`.